fdbclient/NativeAPI.actor.cpp

```diff
@@ -10762,43 +10802,15 @@ Future<bool> DatabaseContext::unblobbifyRange(KeyRange range, Optional<TenantNam
 
 ACTOR Future<Standalone<VectorRef<KeyRangeRef>>> listBlobbifiedRangesActor(Reference<DatabaseContext> cx,
                                                                            KeyRange range,
-                                                                           int rangeLimit,
+                                                                           int rowLimit,
```

Comment:
Nit: Why change this to rowLimit instead of rangeLimit when it gets passed on as rangeLimit to the new getBlobbifiedRanges function? That confused me a bit

Reply:
That's definitely an accident. Also I fixed it for listBlobbifiedRanges() not matching the declaration.